feat(docs): add acl_v4 to best_practices (#608)
emqx/cloud-docs: en_US/best_practices/acl_v4.md

> # Configuring and Using Access Control (ACL) in EMQX Platform v4 Deployments

Reviewer: ```suggestion
# Configure and Use Access Control List (ACL) in EMQX Platform v4 Deployments
```

> Access Control (ACL) is used to manage client permissions for PUBLISH and SUBSCRIBE operations within EMQX Platform.

Reviewer: ```suggestion
Access Control List (ACL) is used to manage client permissions for PUBLISH and SUBSCRIBE operations within EMQX Platform.
```

This page provides detailed information on client ID, username, and quota restrictions, how to enable whitelisting, use Placeholders, and leverage external ACLs for more advanced access control.

> ## Adding Access Control Entries

Reviewer: ```suggestion
## Add Access Control List Entries
```

:::tip NOTE

- The **maximum number of entries** for built-in access control is twice the number of deployment connections, with an upper limit of 100,000. If this limit is exceeded, please use an external ACL.
- Fields can include letters, numbers, and certain special characters (_,-,/,+,#,$,%,@,&, and .), with a maximum length of 128 characters.
:::

Access control can be applied at three levels:

1. **Client ID**: Restrict access via the ClientId field.
2. **Username**: Restrict access via the Username field.
3. **All Users**: Apply access control rules to all users.

In the **Add ACL** dialog, fill in the Client ID or Username, the topic, select whether to allow or deny access, then choose the action (sub/pub/pubsub), and finally click the **Add** button to complete the operation.

> ### Bulk Importing Access Control Entries

Reviewer: ```suggestion
### Bulk Import ACL Entries
```

You can import access control entries in bulk using either of the following methods:

1. You can bulk import access control entries using a provided CSV template (this is not supported for "All Users"). The import fields are as follows:

   - `clientid`: Client ID

   - `username`: Username

   - `topic`: Topic for access control

   - `action`: Action (sub/pub/pubsub)

   - `access`: Allow or deny (allow/deny)

2. To import ACL rules via API, refer to the [API](../api/acl.md) section.

> ## Using Placeholders

Reviewer: ```suggestion
## Use Placeholders
```

In the **Topic** field, you can use the following placeholders, which EMQX Platform will automatically replace with client information during requests:

- `%u`: Username
- `%c`: Client ID

If you want to restrict all users to only subscribe to or publish specific topics, you can add an ACL like this for all users:

- Username `%u`, topic `xx/%u/report`
- Client ID `%c`, topic `xx/%c/report`

![add_acl](./_assets/add_acl_placeholder.png)

## How to Enable Whitelist

:::tip NOTE

- By default, access control operates in **blacklist mode** (i.e., all subscriptions and publish are allowed by default).
- The **matching order** for access control is: All Users ACL -> Username/Client ID ACL.
- The combination of ClientId/Username + Topic is unique, meaning that for the same ClientId/Username + Topic, only the most recent record is valid.
:::

To enable whitelisting (i.e., default deny for all subscriptions and publish) in EMQX Platform, you have the following two options:

> ### 1. Not Using External Access Control

Reviewer: ```suggestion
### 1. Not Use External Access Control
```
